mlnx-toolbox/mlnx-toolbox.py

```diff
@@ -0,0 +1,362 @@
+#! /usr/bin/python
+"""
+using soni_sku_create.py  to create a new SONiC SKU based on a <SKU-DEF>.xml file
```

Comment:
typo: sonic

Reply:
Removed file from Commit not relevant to the SKU create script